src/lib/components/playground/Chat.svelte

```
@@ -1,14 +1,14 @@
 <script lang="ts">
 	import { toast } from 'svelte-sonner';
-
+	import { base } from '$app/paths';
 	import { goto } from '$app/navigation';
 	import { onMount, tick, getContext } from 'svelte';
 
 	import {
 		OLLAMA_API_BASE_URL,
```

ESLint

[error] 'OLLAMA_API_BASE_URL' is defined but never used.

(@typescript-eslint/no-unused-vars)